### DISCUSSION TITLE
Autoreload on raspberry

### DISCUSSION THREAD

**Discussion by pr0phe** [thirtythreeforty/neolink#208] [Q&A]
Hi there, ive installed neolink, its ok, but im unable to run it as service on pi.
I make a file neolink-d.service in /etc/systemd/system with following content:

[Unit]
Description=Neolink deamon
After=network.target

[Service]
Type=notify
ExecStart=neolink rtsp --config neolink.toml
ExecReload=kill -HUP $MAINPID
KillMode=process
Restart=on-failure
RestartSec=5s

[Install]
WantedBy=multi-user.target
, then i add it to autoboot: sudo systemctl enable neolink-d, but it doesnt start.
So whats wrong?

**Comment by QuantumEntangledAndy** [comment]
First check the log

```bash
journalctl -xeu neolink-d.service
```

My guess is though that you need the full path to `neolink.toml`

**Comment by pr0phe** [comment]
сен 06 16:23:14 SHomeSrv2 neolink[29151]: Error: Failed to read "neolink.toml"

![image](https://user-images.githubusercontent.com/66335899/132224160-912f400c-afe5-47c1-bb8d-00570b312187.png)

how to describe this path correctly?
Its a main dir or something else

**Reply by QuantumEntangledAndy** [reply]
Get the full path of the current directory with

```
pwd
```

Prepend that path to file name

**Reply by QuantumEntangledAndy** [reply]
You may want to consider copying the toml to somewhere like /usr/local/etc/neolink.toml

Please do check out our guide to setting up a service in `docs/unix_service.md` of this repo

**Reply by pr0phe** [reply]
thnx
